### PR TITLE
Better notes

### PR DESCRIPTION
**Pull request by padiwik**
See also #1264

### CODE REVIEW

File: client/src/game/ui/notes.ts [hunk 1]
```diff
@@ -104,126 +107,100 @@ export const checkSpecialNote = (card: HanabiCard) => {
...
-  );
+  card.state.noteKnownTrash = checkNoteKeywords([
+    'kt',
+    'trash',
+    'stale',
+    'bad',
+  ], note, fullNote);
+
+  card.state.noteNeedsFix = checkNoteKeywords([
+    'fixme',
```
Comment (Zamiell) [inline client/src/game/ui/notes.ts:126]: small thing - make this one line, and if we add an alias we can split it into separate lines

Comment (padiwik) [reply client/src/game/ui/notes.ts:126]: ok, i was wondering about those

File: client/src/game/ui/notes.ts [hunk 2]
```diff
@@ -104,126 +107,100 @@ export const checkSpecialNote = (card: HanabiCard) => {
...
+  ], note, fullNote);
+
+  card.state.noteFinessed = checkNoteKeywords([
+    'f',
+    'hf',
+    'pf',
+    'gd',
+  ], note, fullNote);
+
+  card.state.noteBlank = checkNoteKeywords([
```
Comment (Zamiell) [inline client/src/game/ui/notes.ts:151]: same here

File: client/src/game/ui/notes.ts [hunk 3]
```diff
@@ -104,126 +107,100 @@ export const checkSpecialNote = (card: HanabiCard) => {
...
+    'blank',
+  ], note, fullNote);
+
+  card.state.noteUnclued = checkNoteKeywords([
+    'unclued',
```
Comment (Zamiell) [inline client/src/game/ui/notes.ts:156]: and here

**Comment by Zamiell** [comment]
thank you!!

**Pull request merged**